src/main/java/com/hillel/items_exchange/model/User.java

```diff
@@ -51,6 +51,6 @@
     @OneToMany(mappedBy = "user", cascade = CascadeType.ALL)
     private List<Phone> phones;
 
-    @OneToMany(mappedBy = "user", cascade = CascadeType.ALL)
+    @OneToMany(mappedBy = "user", cascade = CascadeType.ALL, orphanRemoval = true)
     private List<Child> children;
 }
```

Comment:
@lubchik-14  has changed List of Phones and List of Children to Set, so one of you will have a merge conflict. Please, resolve this issue.

Reply:
Yeah, I saw that. Didn’t affect me